src/Files.App/Styles/ThemedIconSet.xaml

```diff
@@ -0,0 +1,4568 @@
+<!--  Copyright (c) 2024 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ResourceDictionary
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:local="using:Files.App.UserControls">
+
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary Source="ms-appx:///UserControls/ThemedIcon/ThemedIcon.xaml" />
+		<ResourceDictionary>
+
+			<!--  Empty  -->
+			<Style
+				x:Name="TIcon_Empty"
+				BasedOn="{StaticResource DefaultThemedIconStyle}"
+				TargetType="local:ThemedIcon">
+
+				<Setter Property="Width" Value="16" />
+				<Setter Property="Height" Value="16" />
+
+				<Setter Property="OutlineIconData" Value="" />
+				<Setter Property="FilledIconData" Value="" />
+
+				<Setter Property="Layers">
+					<Setter.Value>
+						<local:ThemedIconLayers>
+							<!--  Place lower layers first in the list  -->
+							<local:ThemedIconLayer LayerType="Alt" PathData="" />
+							<local:ThemedIconLayer LayerType="Base" PathData="" />
+							<local:ThemedIconLayer LayerType="Accent" PathData="" />
+							<local:ThemedIconLayer LayerType="AccentContrast" PathData="" />
+						</local:ThemedIconLayers>
+					</Setter.Value>
+				</Setter>
+			</Style>
+
+			<!--#region  Common Icons-->
+			<Style
+				x:Name="TIcon_NavForward"
```

Comment:
I would suggest it be ThemedIcon**s**.NavForward so as not to be confused by any of the ThemedIcon properties

Reply:
`App.Icons.NavForward` to keep consistency with `App.Themes.*`

Reply:
`ThemedIcons.NavForward` & `App.Icons.NavForward` are fine with me.

Reply:
App.ThemedIcons.NavForward is a good way to do it.

It's clear which type of icon control is used, and the App, follows other conventions.

Reply:
You mean, like PathIcons? Shouldn't they have all different names even among other types?

Reply:
There may be a mix of Icons as you start to replace older ones with ThemedIcon versions.

PathIcons, OpacityIcons, and there may be FontIcons and SymbolIcons there alongside ThemeIcons.

Reply:
ThemedIcons can be `App.Icons.*` since others already have other names.

![image](https://github.com/files-community/Files/assets/62196528/48d2ad92-95ff-4650-adca-3a6a94fb3c9c)

Reply:
I would still opt for App.ThemedIcons.LayoutTiles

and we only append a size when there are larger icons than the 16x16 ones

Reply:
@0x5bfa I pushed a commit. Make sure to pull the commit before making any other changes.